### PR TITLE
Update Autofac deps to latest major version

### PR DESCRIPTION
**Pull request by alsami**
* Update `Autofac.Extensions.DependencyInjection` to version `6.0.0`
* Update `Autofac.Multitenant` to version `5.0.0`
* Bump version-prefix to `3.0.0` since this now requires `Autofac` >= `5.0.0`

### CODE REVIEW

**Comment by alsami** [comment]
@tillig `Autofac.Multitenant` must be merged and published (5.0.0) first for this :)

File: src/Autofac.Integration.AspNetCore.Multitenant/Autofac.Integration.AspNetCore.Multitenant.csproj [hunk 1]
```diff
@@ -37,7 +37,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="5.0.1-develop-00435" />
+    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="6.0.0" />
     <PackageReference Include="Autofac.Multitenant" Version="5.0.0-develop-00267" />
```
Comment (tillig) [inline src/Autofac.Integration.AspNetCore.Multitenant/Autofac.Integration.AspNetCore.Multitenant.csproj]:
```suggestion
     <PackageReference Include="Autofac.Multitenant" Version="5.0.0" />
```

**Review by tillig — CHANGES_REQUESTED** [review]
Ready for Autofac.Multitenant 5.0.0!

**Comment by alsami** [comment]
> Ready for Autofac.Multitenant 5.0.0!

Done, done and done! :)

**Review by tillig — APPROVED** [review]
🎉

**Pull request merged**